apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/TransactionBroadcastedContent.tsx

```diff
@@ -1,25 +1,27 @@
 import React from "react";
 import { Trans } from "react-i18next";
-import { SwapSelectorStateType } from "@ledgerhq/live-common/exchange/swap/types";
 import Box from "~/renderer/components/Box";
 import { Separator } from "~/renderer/screens/exchange/Swap2/Form/Separator";
 import Button from "~/renderer/components/Button";
 import SwapCompleted from "~/renderer/screens/exchange/Swap2/Form/ExchangeDrawer/SwapCompleted";
 import { useGetSwapTrackingProperties } from "~/renderer/screens/exchange/Swap2/utils";
 import TrackPage from "~/renderer/analytics/TrackPage";
+import { Currency } from "@ledgerhq/types-cryptoassets";
+import SellCompleted from "~/renderer/screens/exchange/Sell/SellCompleted";
 
 type TransactionBroadcastedContentProps = {
-  swapId: string;
+  swapId?: string;
+  isSell?: boolean;
```

Comment:
Can we do something more consistent?
```suggestion
  quoteId?: string;
  mode: "sell" | "swap" ;
```

Reply:
Since there is no `sellId`, i have added the `isSell` property. This flow is very twisted, having a `mode` would be ideal but I would have a lot to refactor. Don't worry, the logic is pretty sound, the `isSell` will ever be true only if the transaction is a sell.